cmsdb/processes/hh2bbvv.py

```diff
@@ -7,68 +7,61 @@
-    "hh_ggf_kl2p45_kt1_hbb_hvv2l2nu", "hh_ggf_kl5_kt1_hbb_hvv2l2nu",
-    # HH -> bbVV(2l2v), vbf
-    "hh_vbf_hbb_hvv2l2nu", "hh_vbf_kv0p5_k2v1_kl1_hbb_hvv2l2nu", "hh_vbf_kv1p5_k2v1_kl1_hbb_hvv2l2nu",
-    "hh_vbf_kv1_k2v0_kl1_hbb_hvv2l2nu", "hh_vbf_kv1_k2v1_kl0_hbb_hvv2l2nu", "hh_vbf_kv1_k2v1_kl1_hbb_hvv2l2nu",
-    "hh_vbf_kv1_k2v1_kl2_hbb_hvv2l2nu", "hh_vbf_kv1_k2v2_kl1_hbb_hvv2l2nu",
-    # HH -> bbWW, ggf
-    "hh_ggf_hbb_hww", "hh_ggf_kl0_kt1_hbb_hww", "hh_ggf_kl1_kt1_hbb_hww",
-    "hh_ggf_kl2p45_kt1_hbb_hww", "hh_ggf_kl5_kt1_hbb_hww",
-    # HH -> bbWW, vbf
-    "hh_vbf_hbb_hww", "hh_vbf_kv0p5_k2v1_kl1_hbb_hww", "hh_vbf_kv1p5_k2v1_kl1_hbb_hww",
-    "hh_vbf_kv1_k2v0_kl1_hbb_hww", "hh_vbf_kv1_k2v1_kl0_hbb_hww", "hh_vbf_kv1_k2v1_kl1_hbb_hww",
-    "hh_vbf_kv1_k2v1_kl2_hbb_hww", "hh_vbf_kv1_k2v2_kl1_hbb_hww",
-    # HH -> bbWW(qqlv), ggf
-    "hh_ggf_hbb_hwwqqlnu", "hh_ggf_kl0_kt1_hbb_hwwqqlnu", "hh_ggf_kl1_kt1_hbb_hwwqqlnu",
-    "hh_ggf_kl2p45_kt1_hbb_hwwqqlnu", "hh_ggf_kl5_kt1_hbb_hwwqqlnu",
-    # HH -> bbWW(lvlv), ggf
-    "hh_ggf_hbb_hww2l2nu", "hh_ggf_kl0_kt1_hbb_hww2l2nu", "hh_ggf_kl1_kt1_hbb_hww2l2nu",
-    "hh_ggf_kl2p45_kt1_hbb_hww2l2nu", "hh_ggf_kl5_kt1_hbb_hww2l2nu",
-    # HH -> bbWW(qqlv), vbf
-    "hh_vbf_hbb_hwwqqlnu", "hh_vbf_kv0p5_k2v1_kl1_hbb_hwwqqlnu", "hh_vbf_kv1p5_k2v1_kl1_hbb_hwwqqlnu",
-    "hh_vbf_kv1_k2v0_kl1_hbb_hwwqqlnu", "hh_vbf_kv1_k2v1_kl0_hbb_hwwqqlnu", "hh_vbf_kv1_k2v1_kl1_hbb_hwwqqlnu",
-    "hh_vbf_kv1_k2v1_kl2_hbb_hwwqqlnu", "hh_vbf_kv1_k2v2_kl1_hbb_hwwqqlnu",
-    # HH -> bbWW(lvlv), vbf
-    "hh_vbf_hbb_hww2l2nu", "hh_vbf_kv0p5_k2v1_kl1_hbb_hww2l2nu", "hh_vbf_kv1p5_k2v1_kl1_hbb_hww2l2nu",
-    "hh_vbf_kv1_k2v0_kl1_hbb_hww2l2nu", "hh_vbf_kv1_k2v1_kl0_hbb_hww2l2nu", "hh_vbf_kv1_k2v1_kl1_hbb_hww2l2nu",
-    "hh_vbf_kv1_k2v1_kl2_hbb_hww2l2nu", "hh_vbf_kv1_k2v2_kl1_hbb_hww2l2nu",
-    "hh_ggf_hbb_hzz",
-    # HH -> bbZZ, ggf
-    "hh_ggf_hbb_hzz", "hh_ggf_kl0_kt1_hbb_hzz", "hh_ggf_kl1_kt1_hbb_hzz",
-    "hh_ggf_kl2p45_kt1_hbb_hzz", "hh_ggf_kl5_kt1_hbb_hzz",
-    # HH -> bbZZ, vbf
-    "hh_vbf_hbb_hzz", "hh_vbf_kv0p5_k2v1_kl1_hbb_hzz", "hh_vbf_kv1p5_k2v1_kl1_hbb_hzz",
-    "hh_vbf_kv1_k2v0_kl1_hbb_hzz", "hh_vbf_kv1_k2v1_kl0_hbb_hzz", "hh_vbf_kv1_k2v1_kl1_hbb_hzz",
-    "hh_vbf_kv1_k2v1_kl2_hbb_hzz", "hh_vbf_kv1_k2v2_kl1_hbb_hzz",
-    # HH -> bbZZ(lvlv), ggf
-    "hh_ggf_hbb_hzz2l2nu", "hh_ggf_kl0_kt1_hbb_hzz2l2nu", "hh_ggf_kl1_kt1_hbb_hzz2l2nu",
-    "hh_ggf_kl2p45_kt1_hbb_hzz2l2nu", "hh_ggf_kl5_kt1_hbb_hzz2l2nu",
-    # HH -> bbZZ(lvlv), vbf
-    "hh_vbf_hbb_hzz2l2nu", "hh_vbf_kv0p5_k2v1_kl1_hbb_hzz2l2nu", "hh_vbf_kv1p5_k2v1_kl1_hbb_hzz2l2nu",
-    "hh_vbf_kv1_k2v0_kl1_hbb_hzz2l2nu", "hh_vbf_kv1_k2v1_kl0_hbb_hzz2l2nu", "hh_vbf_kv1_k2v1_kl1_hbb_hzz2l2nu",
-    "hh_vbf_kv1_k2v1_kl2_hbb_hzz2l2nu", "hh_vbf_kv1_k2v2_kl1_hbb_hzz2l2nu",
+    f"hh_ggf_hbb_hvv{vv}{params}"
+    for params in ["", "_kl0_kt1", "_kl1_kt1", "_kl2p45_kt1", "_kl5_kt1"]
```

Comment:
Not really a comment, but the ggf (and vbf below) kappa "params" seem repeated, it could make sense to write them only once.

Reply:
I'll keep that in mind for future PRs :+1: